dask/dataframe/core.py

```diff
@@ -99,6 +99,9 @@ def __dask_keys__(self):
     def __dask_tokenize__(self):
         return self._name
 
+    def __dask_layers__(self):
+        return [self.key]
```

Comment:
In Array, this is a 1-element tuple. Does the exact type matter?

Reply:
I've normalized everyone to use tuples for now.